.github/workflows/gatsby-build.yml

```diff
@@ -10,6 +10,9 @@ jobs:
         with:
           node-version: "14.16.0"
       
-      - run: npm ci
-      - run: npm install --global gatsby-cli
-      - run: source ./start.sh && gatsby build --verbose
\ No newline at end of file
+      - run: |
+          npm ci
+          npm install -g typescript
```

Comment:
I think the type script version is specified in the package json. Can can we use the local typescirpt instead of the global one (installed with the npm ci).